### PR TITLE
Add ActionDispatch::SystemTestCase#fill_in_rich_text_area

### PR DESCRIPTION
**Pull request by georgeclaghorn**
`fill_in_rich_text_area` locates a Trix editor and fills it in with the given HTML:

```ruby
# <trix-editor id="message_content" ...></trix-editor>
fill_in_rich_text_area "message_content", with: "Hello world!"

# <trix-editor placeholder="Your message here" ...></trix-editor>
fill_in_rich_text_area "Your message here", with: "Hello world!"

# <trix-editor aria-label="Message content" ...></trix-editor>
fill_in_rich_text_area "Message content", with: "Hello world!"

# <input id="trix_input_1" name="message[content]" type="hidden">
# <trix-editor input="trix_input_1"></trix-editor>
fill_in_rich_text_area "message[content]", with: "Hello world!"
```

### CODE REVIEW

**Review by eileencodes — APPROVED** [review]
Looks good 👍

File: actiontext/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+   # <trix-editor placeholder="Your message here" ...></trix-editor>
+   fill_in_rich_text_area "Your message here", with: "Hello world!"
+
+   # <trix-editor aria-label="Message content" ...></trix-editor>
+   fill_in_rich_text_area "Message content", with: "Hello world!"
+
+   # <input id="trix_input_1" name="message[content]" type="hidden">
+   # <trix-editor input="trix_input_1"></trix-editor>
+   fill_in_rich_text_area "message[content]", with: "Hello world!"
+   ```
```
Comment (javan) [inline actiontext/CHANGELOG.md]: Could reinforce that you're filling in HTML, not plain text: `with: "Hello <em>world!</em>"`

**Review by javan — APPROVED** [review]
💰📝

**Comment by twalpole** [comment]
Nice to see someone using Capybaras custom selector types.  Just to point out that most of Capybaras built-in selectors only apply the locator conditions if it's not nil, which allows for using without the locator if there is only one matching field within the current scope.  Might be a useful addition here

File: actiontext/lib/action_text/system_test_helper.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    #   fill_in_rich_text_area "Your message here", with: "Hello <em>world!</em>"
+    #
+    #   # <trix-editor aria-label="Message content" ...></trix-editor>
+    #   fill_in_rich_text_area "Message content", with: "Hello <em>world!</em>"
+    #
+    #   # <input id="trix_input_1" name="message[content]" type="hidden">
+    #   # <trix-editor input="trix_input_1"></trix-editor>
+    #   fill_in_rich_text_area "message[content]", with: "Hello <em>world!</em>"
+    def fill_in_rich_text_area(locator, with:)
+      page.execute_script(<<~JS, find(:rich_text_area, locator).native, with.to_s)
```
Comment (twalpole) [inline actiontext/lib/action_text/system_test_helper.rb]: There should be no need to call `.native` here, Capybara will convert to the correct parameter for the driver, and with Capybara 3.2+ you can call execute_script on the element itself which will pass the element as the first argument

    page.find(:rich_text_area, locator).execute_script(<<~JS, with.to_s)
      ...

**Comment by georgeclaghorn** [comment]
Thanks, @twalpole! I made those changes in a0b1029 and 339be65.

**Pull request merged**